static/css/configuration.less

```diff
@@ -90,3 +90,12 @@ dl.horizontal {
     }
   }
 }
+
+.mb-50 {
+  margin-bottom: 50px;
+}
+
+.page.configuration .label {
```

Comment:
I think this will affect other configuration pages too. Maybe the selector has to be more specific, like `.upgrade-config .label`.